strawberry/auto.py

```diff
@@ -1,4 +1,2 @@
-import sentinel
-
-
-auto = sentinel.create("auto")
+class auto:
+    pass
```

Comment:
shall we prevent people from instanciating auto? :)

Reply:
Sure! I can add a `__new__` that raises an exception if called.